mPower2/MotorControl/MCTActiveStepViewController.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  MCTActiveStepViewController.swift
+//  MotorControl
+//
+//  Copyright © 2018 Sage Bionetworks. All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without modification,
+// are permitted provided that the following conditions are met:
+//
+// 1.  Redistributions of source code must retain the above copyright notice, this
+// list of conditions and the following disclaimer.
+//
+// 2.  Redistributions in binary form must reproduce the above copyright notice,
+// this list of conditions and the following disclaimer in the documentation and/or
+// other materials provided with the distribution.
+//
+// 3.  Neither the name of the copyright holder(s) nor the names of any contributors
+// may be used to endorse or promote products derived from this software without
+// specific prior written permission. No license is granted to the trademarks of
+// the copyright holders even if such marks are included in this software.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+// ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE
+// FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+// DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+// SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+// CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+// OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+// OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+
+import Foundation
+
+open class MCTActiveStepViewController : RSDActiveStepViewController {
+    
+    @IBOutlet weak var restartButton: RSDRoundedButton!
+    
+    /// What unit should the unit label display to the user?
+    @IBInspectable
+    open var unitLabelText : String? = nil {
+        didSet {
+            self.unitLabel?.text = unitLabelText
+        }
+    }
+    
+    /// This class overrides `didSet` to update the `countdownLabel` to the new value formatted as a time interval in seconds.
+    /// The `countdownFormatter` is used to format the string derived from this time interval.
+    override open var countdown: Int {
+        didSet {
+            self.countdownLabel?.text = countdownFormatter.string(from: TimeInterval(countdown))
+        }
+    }
+    
+    /// Formatter for the countdown label.
+    /// Overriden to not display minutes when the minutes place would be 0.
+    override lazy open var countdownFormatter : DateComponentsFormatter = {
+        let formatter = DateComponentsFormatter()
+        formatter.allowedUnits = countdown < 60 ? [.second] : [.minute, .second]
```

Comment:
Actually, you only ever want seconds for this UI design. We are using a custom subclass so that we know that the UI will match the designs for the cases where we are using it, but should someone else try to use this same design (copy/paste) for a step that is more than 60 seconds, this will format incorrectly when really they want "90 seconds remaining"